devtools/conda-recipe/meta.yaml

```diff
@@ -21,6 +21,8 @@ requirements:
     - lxml
     - openmoltools >=0.7.3
     - parmed
+    - pylab
```

Comment:
When I have matplotlib there I get this error in the Travis test:
```
Test FragmentSampler runs for 10 iterations with no failures ... QXcbConnection: Could not connect to display 
/home/travis/build.sh: line 59:  1702 Aborted                 (core dumped) nosetests -vv --nocapture --with-timer $PACKAGENAME
```
I get pylab is a module and not the package, but that is what is working...